cmd/botkube/main.go

```diff
@@ -216,8 +216,9 @@ func run(ctx context.Context) error {
 	}
 
 	var (
-		notifiers []notifier.Notifier
-		bots      = map[string]bot.Bot{}
+		botNotifiers  []notifier.Bot
+		sinkNotifiers []notifier.Sink
+		bots          = map[string]bot.Bot{}
```

Comment:
`botNotifiers []notifier.Bot` is redundant aa we have this bot map. This map type can be changed to

```
bots = map[string]notifier.Bot{}
```

and essentially pass it everywhere instead of the slice. But we can do it later - can you change the type and add this TODO at least? Thanks!

```suggestion
		// TODO: Use bots everywhere instead of `botNotifiers` slice
		bots          = map[string]notifier.Bot{}
```

Reply:
`bot.Bot` refers to bot implementation while `bot.Notifier` is for notification. When I compare with main, we have `bots` and `notifiers`

Reply:
Already discussed f2f 🙂 tl;dr for others: we use bots map for `sendHelp` where we need only the `SendMessageToAll`